README.md

```diff
@@ -57,3 +57,7 @@ generated files which should appear in the ./static folder**
 1. serve the site locally by running from the command line:
 
     `hugo serve`
+
+#### If using Rstudio 
```

Comment:
So this step should actually not be needed any more, as I simplified things a little bit. Just knitting the .Rmd files to .md files should be enough, and I would no longer expect to see .html files committed to the repo. Is there a reason you thought this extra step was necessary?

Reply:
Ok that's fine, I think it was because I've been doing everything through R studio and had used this way but happy to keep under what you've done / think is necessary :)